pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -390,10 +395,24 @@ public String getGlobalZookeeperServers() {
         return globalZookeeperServers;
     }
 
+    @Deprecated
     public void setGlobalZookeeperServers(String globalZookeeperServers) {
         this.globalZookeeperServers = globalZookeeperServers;
     }
 
+    public String getConfigurationStoreServers() {
+        if (this.configurationStoreServers == null || this.configurationStoreServers.isEmpty()) {
+            // If the configuration is not set, assuming that the globalZK is not enabled and all data is in the same
+            // ZooKeeper cluster
+            return this.getZookeeperServers();
```

Comment:
you should call `return this.getGlobalZookeeperServers()`, otherwise it breaks BC.